public/app/core/components/AppChrome/News/NewsContainer.tsx

```diff
@@ -36,12 +36,12 @@ export function NewsContainer({ className }: NewsContainerProps) {
                 <img src="public/img/grot-news.svg" alt="Grot reading news" />
               </a>
               <div className={styles.actions}>
-                <Button
-                  icon="times"
+                <IconButton
+                  name="times"
                   variant="secondary"
-                  fill="text"
                   onClick={onToggleShowNewsDrawer}
-                  aria-label={selectors.components.Drawer.General.close}
+                  data-testid={selectors.components.Drawer.General.close}
+                  tooltip={t(`drawer.close`, 'Close Drawer')}
```

Comment:
```suggestion
                  tooltip={t(`news.drawer.close`, 'Close Drawer')}
```

To translate something, please see the [contribution guidelines](https://github.com/grafana/grafana/blob/main/contribute/internationalization.md#how-to-add-a-new-translation-phrase) for it. You should now run `yarn i18n:extract` so that these get extracted into translation files so that they can be translated 😄

Reply:
My bad. Apologies for that.

I have updated the translation files.

Reply:
No problem at all 😄